crates/core/src/factories/mem_transport/test.rs

```diff
@@ -367,9 +399,11 @@ async fn transport_preflight_reject() {
     );
 
     let t1 = gen_tx(h1.clone()).await;
+    t1.register_space_handler(TEST_SPACE_ID, h1.clone());
```

Comment:
Space handler required in tests where it previously wasn't required because the space handler is used when checking for blocks.